src/UrlStore.php

```diff
@@ -4,13 +4,14 @@
 
 namespace Horde\Core;
 
-use Horde;
 use Horde_Url;
+use Horde;
+Use Horde_Registry;
 
-class UrlStore
+class UrlStore 
 {
     public function getInitialPage(string $app, bool $full): Horde_Url
     {
-        return Horde::Url($this->registry->getInitialPage($app), $full);
+        return Horde::Url($this->registry->getInitialPage($app, $full));
```

Comment:
This change is wrong.

Reply:
what should I write here or what is wrong specifically?